Sources/AppStoreConnectCLI/Commands/Certificates/ListDownloadCertificatesCommand.swift

```diff
@@ -69,21 +55,18 @@ struct ListDownloadCertificates: CommonParsableCommand {
 
         if let downloadPath = downloadPath {
             try certificates.forEach { (certificate: Certificate) in
-                guard let content = certificate.content else {
-                    throw CommandError.invalidContent
+                guard let content = certificate.content,
+                    let data = Data(base64Encoded: content) else {
+                    throw CertificatesError.noContent
```

Comment:
Don't know about the rest of the team but I can find the following formatting confusing as the indentation for the `guard let` bindings is the same for the `throw`

```suggestion
                guard 
                    let content = certificate.content,
                    let data = Data(base64Encoded: content)
                else {
                    throw CertificatesError.noContent
```

Reply:
I've done this in the past. I think it helps with readability too.